### DISCUSSION TITLE
Change TikTok Front Page UI Button Colors to Prevent OLED Screen Burn-In

### DISCUSSION THREAD

**Discussion by Imikeeee** [dayanch96/BHTikTok-Plus#27] [Ideas]
As smartphones with OLED screens become increasingly popular, there's a growing concern about screen burn-in, which can negatively impact the user experience. To address this, I propose a simple but effective change for TikTok: adjust the front page UI button colors to prevent OLED screen burn-in.

Implementation:

1. Color Transition: Rotate the UI button colors slightly over time. This can be achieved with a subtle animation that gradually changes the button colors. This approach helps distribute the stress on individual pixels and reduces the risk of burn-in.
2. Color Testing: Prior to implementation, conduct thorough testing on various OLED devices to ensure that the proposed changes effectively mitigate screen burn-in without compromising the user experience.
3. User Customization: To accommodate user preferences, provide an option in the app's settings for users to choose between the original color scheme and the burn-in prevention color scheme.

**Comment by dayanch96** [comment]
Hi. Sorry, i don’t have any ios device to work with right now
